clients/email-template-client/src/openapi.d.ts

```diff
@@ -9,6 +9,37 @@ import type {
 
 declare namespace Components {
     namespace Schemas {
+        export interface AsyncEmailTemplateResponse {
+            /**
+             * Job ID of the email template that is requested to replace and generate docs
+             * example:
+             * 8c086140-f33e-4bb7-a993-50c0f2402c7b
+             */
+            job_id: string;
+            status: "STARTED" | "PROCESSING" | "SUCCESS" | "FAILED";
+            /**
+             * Error message
+             */
+            message?: string;
+            /**
+             * Progress count of the documents that are needed to generate
+             */
+            doc_progress_count?: {
+                /**
+                 * Total count
+                 * example:
+                 * 10
+                 */
+                total?: number;
+                /**
+                 * Completed count
+                 * example:
+                 * 5
+                 */
+                completed?: number;
```

Comment:
Would be create for UI if these properties would be non-nullable. If the `doc_progress_count` is defined then I would assume that these values are always defined also.